evals/benchmark/config.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2024 Intel Corporation

```

Comment:
use this copyright

# Copyright (C) 2024 Intel Corporation
# SPDX-License-Identifier: Apache-2.0